tomcat-8.5/jdk-11-ubuntu-18.04.gradle

```diff
@@ -11,7 +11,7 @@ ext {
             flavor : 'jdk',
             version: [
                     major: 11,
-                    update: 9
+                    update: 10
```

Comment:
minor does not need to be set? I see it was't present before either, which is unexpected.

Reply:
OK, I checked other files and it isn't set anywhere, so you can merge.

Reply:
minor version is hardcoded it seems. As long as we dont need it, I suppose we can leave it as is.